R/gsplot-class.R

```diff
@@ -14,7 +14,7 @@ gsplot <- function(x = NULL, ...) UseMethod("gsplot")
 #' @rdname gsplot
 #' @export
 gsplot.default <- function(...) {
-  object <- gsplot(list())
+  object <- gsplot(list(global=list('config'=list(frame.plot=TRUE))))
```

Comment:
@jiwalker-usgs I'd like to hear where you think these defaults should live. Obviously, not here. But, I can't add them to default.yaml w/o hacks